### ISSUE TITLE
Add number pad

### ISSUE THREAD

**Issue by Arthur-McArthur** [tkainrad/keycombiner#97]
Would be good to be able to use shortcuts from the number pad

**Comment by tkainrad** [comment]
You can already include number pad keys in your shortcuts!

See [this wiki page](https://github.com/tkainrad/keycombiner/wiki/Valid-Key-Combinations) for a full list of available keys. Number keys are <kbd>numpad0</kbd>-<kbd>numpad9</kbd>.

**Comment by Arthur-McArthur** [comment]
That's helpful, thanks. It would be good to be able to visualize them as well. The visual keyboard only shows a tenkeyless design, which makes it more difficult to get the complete picture at a glance.

**Comment by TheHipshaker** [comment]
Are there any plans to incorporate all the numpad keys? In particular we would like to use the divide (/) key to collapse entire tree views for example...

**Comment by FreakinRocket** [comment]
I also agree that adding the other numpad keys such as -+/* and enter would be fantastic. Many shortcuts require these numpad keys
